### DISCUSSION TITLE
pocketbase as db without autorisation/autentification

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#1133] [Q&A] [answered]
Hello!
if I don’t need authorization or registration for the site, but just as a kind of data storage, do I understand correctly that I can simply log in as a database administrator in the endpoint (sveltekit) and make all requests as an administrator?

**Comment by ganigeorgiev** [comment, marked as answer]
> do I understand correctly that I can simply log in as a database administrator in the endpoint (sveltekit) and make all requests as an administrator?

Yes. Admins have access to everything and API rules don't apply to them.

Just make sure that your files are [server-only modules](https://kit.svelte.dev/docs/server-only-modules) (eg. has `.server` in the filename) to avoid accidentally leaking your admin password to the client.
